src/microbe_stage/BiomeConditions.cs

```diff
@@ -320,22 +338,34 @@ public void ApplyLongTermCompoundChanges(Biome biomeDetails, Dictionary<Compound
     ///   Modifies the long-term amount of a compound in this biome. This is preferable to directly modifying
     ///   <see cref="ChangeableCompounds"/>. This is only usable for compounds that don't vary along an in-game day.
     /// </summary>
-    /// <param name="compound">The compound to modify</param>
-    /// <param name="newValue">New value to set</param>
-    public void ModifyLongTermCondition(Compound compound, BiomeCompoundProperties newValue)
+    /// <param name="compound">
+    ///   The compound to modify
```

Comment:
Short param definitions shouldn't be split across multiple lines.

```suggestion
    ///   <param name="compound">The compound to modify</param>
```